New_PlatformGame/Assets/1. Script/MissileSpawn.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MissileSpawn : MonoBehaviour
+{
+    private Vector3 CameraPositionTopRight;
+    private Transform transform;
```

Comment:
```suggestion
    private Vector3 CameraPositionTopRight;
    private Transform transform;
```
`private변수 앞에는 _로 시작한다.` 라는 코드 컨벤션을 지키지 않았어요.
`변수 이름은 소문자로, 메서드와 클래스 이름은 대문자로 시작한다.` 라는 컨벤션도 지켜지지 않았습니다.